airbyte-integrations/connectors/destination-bigquery/metadata.yaml

```diff
@@ -26,7 +26,7 @@ data:
         message: >
           This version introduces [Destinations V2](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2/#what-is-destinations-v2), which provides better error handling, incremental delivery of data for large syncs, and improved final table structures. To review the breaking changes, and how to upgrade, see [here](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2/#quick-start-to-upgrading). These changes will likely require updates to downstream dbt / SQL models, which we walk through [here](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2/#updating-downstream-transformations).
 
-          Selecting `Upgrade` will upgrade **all** connections using this destination at their next sync. You can manually sync existing connections prior to the next scheduled sync to start the upgrade early.
+          Selecting `Upgrade` will upgrade **all** connections using this destination at their next sync. You can manually sync existing connections prior to the next scheduled sync to start the upgrade early.  **DO NOT UPGRADE UNLESS YOU HAVE RUN A TEST MIGRATION AS OUTLINED [HERE](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2/#testing-destinations-v2-for-a-single-connection)**.
```

Comment:
I would put the big warning of "don't push the button until xyz" before the description of what happens when you decide to actually push the upgrade button, but otherwise 👍🏻 from me

Reply:
If we do that, I think it would also be fair to remove the all caps.